feature/day4/Task7.js

```diff
@@ -0,0 +1,14 @@
+// 7. Write a program to find index of duplicate elements in an array
+let arr = [10, 20, 30, 10, 30, 40, 50, 60, 10, 60];
+let index = [];
+var count = 0;
```

Comment:
what is count doing in this program?
remove count and without using Set() solve the problem